misc/config_definitions.json

```diff
@@ -4948,7 +4955,10 @@
                 }
             ],
             "validate": {
-                "value.*.authalgo": "in:MD5,SHA"
+                "value.*.authalgo": "in:MD5,SHA,SHA-224,SHA-256,SHA-384,SHA-512"
+            }
+            "validate": {
```

Comment:
woopsie

Reply:
@hanserasmus isnt this wrong? dual `validate`

Reply:
Don't think so?. We need to test for both SHA2 capabalities, as well as new crypto-algorithm capabilities.

Reply:
You need to put them both in the same validate (otherwise you only get one as it overwrites the other key)